frontend/src/pages/SwitchIntergation/SwitchIntergation.tsx

```diff
@@ -0,0 +1,374 @@
+														to={`https://www.highlight.io/docs/general/integrations/${integration.key}-integration`}
+														target="_blank"
+														size="xSmall"
+														kind="secondary"
+														emphasis="low"
+														iconLeft={
+															<IconSolidDocument />
+														}
+													>
+														Docs
+													</LinkButton>
+												</Stack>
+												<Stack gap="1">
+													<Text
+														color="secondaryContentText"
+														size="xSmall"
+														cssClass="ml-2"
+													>
+														Website
+													</Text>
+													<LinkButton
+														trackingId="intergation_type-docs-link"
+														//for now redirect to docs. TO DO update it to integration url
+														to={`https://www.highlight.io/docs/general/integrations/${integration.key}-integration`}
+														target="_blank"
+														size="xSmall"
+														kind="secondary"
+														emphasis="low"
+														iconLeft={
+															<IconSolidGlobe />
+														}
+													>
+														{integration.key}
+														.com
+													</LinkButton>
+												</Stack>
+											</Stack>
+											<Box cssClass="justify-self-end">
+												{integration.defaultEnable ? (
+													<Button
+														onClick={() => {
+															setShowConfiguration(
+																false,
+															)
+															setShowDeleteConfirmation(
+																true,
+															)
+														}}
+													>
+														Disconnect
+													</Button>
+												) : (
+													<Button
+														iconRight={
+															<IconSolidArrowSmRight />
+														}
+														onClick={(e) => {
+															e.preventDefault()
+															setShowConfiguration(
+																true,
+															)
+														}}
+													>
+														Connect
+													</Button>
+												)}
+											</Box>
+										</Box>
+									</BorderBox>
+								)
+							})}
+							<Box mt="24">
+								<Tabs<IntegrationSettingsTabs>
+									selectedId={
+										(params?.get('tab') ||
+											IntegrationSettingsTabs.Description) as IntegrationSettingsTabs
+									}
+									onChange={(id) => {
+										navigate(
+											`${location.pathname}?tab=${id}`,
+										)
+									}}
+								>
+									<Tabs.List>
+										<Tabs.Tab
+											id={
+												IntegrationSettingsTabs.Description
+											}
+										>
+											Description
+										</Tabs.Tab>
+										<Tabs.Tab
+											id={
+												IntegrationSettingsTabs.Configure
+											}
+										>
+											Configure
+										</Tabs.Tab>
+									</Tabs.List>
+									<Box mt="24">
+										<Tabs.Panel
+											id={
+												IntegrationSettingsTabs.Description
+											}
+										>
+											<Stack>
+												{/* TO DO  Need update this with real description*/}
```

Comment:
for the descriptions, a starting point for the blurbs can be found on our docs: https://www.highlight.io/docs/general/integrations/overview

Reply:
see the individual pages. for instance, the vercel description can have the notes from the page https://www.highlight.io/docs/general/integrations/vercel-integration